pusher/main.go

```diff
@@ -351,8 +354,14 @@ func Pusher(etcdConn *clientv3.Client, root, etcdRoot string) {
 		if !content[k].isDir {
 			_, err = etcdConn.Put(ctx, fileP, string(content[k].content))
 		}
-		if err != nil {
-			log.Fatalf("error when setting znode >%s(%s + %s)<. Config server will be inconsistent: %s",
+		// if the key is too large we ignore the error
+		// https://etcd.io/docs/v3.3/upgrades/upgrade_3_2/#changed-maximum-request-size-limits-3210
+		// error msg in the document is slightly off, so we define it here.
+		// ErrRequestTooLarge comes from go.etcd.io/etcd v3.3.27+incompatible, but we are only using string so it should be find
+		if err != nil && (strings.HasPrefix(err.Error(), clientTooLargeErr) || err.Error() == rpctypes.ErrRequestTooLarge.Error()) {
+			fmt.Printf("file %s\n is too large, error: %s\n", fileP, err.Error())
```

Comment:
Do we need to log it?
If so, we can use  `log.Printf` instead.

Reply:
updated